tracee-ebpf/main.go

```diff
@@ -404,6 +407,11 @@ func prepareOutput(outputSlice []string, containerMode bool) (tracee.OutputConfi
 		}
 	}
 
+	if printerKind == "table" ||
+		true { // XXX: Remove this line once tracee-rules signatures have been updated to take raw args by default into account (see #1123, libbpfgo#88)
```

Comment:
With this line, there is no way to have raw arguments set (ParseArguments will always be true). Am I wrong?
I think this line should be removed, and the entrypoint should be changed (for now) to have parseArguments option set

Reply:
You're right - I've updated entrypoint.sh which makes a lot more sense.